components/sensor/total_energy.rst

```diff
@@ -0,0 +1,81 @@
+Total Daily Energy Sensor
+=========================
+
+.. seo::
+    :description: Instructions for setting up sensors that track the total energy usage and accumulate the power usage. It can be reset at any given time interval using automation.
+    :image: sigma.png
+
+The ``total_energy`` sensor is a helper sensor that can use the energy value of
```

Comment:
There is no `total_energy` sensor, just `total_daily_energy` - Please modify the existing `total_daily_energy.rst` file instead of creating a new one.

Reply:
oh was a mistake.. good eyes ;).. thx